src/screens/time/Punch.tsx

```diff
@@ -187,22 +238,24 @@ class Punch extends React.Component<PunchProps, PunchState> {
   };
 
   onPressPunchButton = () => {
-    const {punchCurrentDateTime, savedNote} = this.props;
-    if (punchCurrentDateTime !== undefined) {
+    const {punchCurrentDate, punchCurrentTime, savedNote} = this.props;
+    if (punchCurrentDate !== undefined) {
       const punchRequest: PunchRequest = {
         timezoneOffset: getCurrentTimeZoneOffset(),
-        note: savedNote ? savedNote : undefined,
-        datetime: getDateSaveFormatFromDateObject(punchCurrentDateTime),
```

Comment:
I think we can remove `getDateSaveFormatFromDateObject` method if not using anymore